x-pack/plugins/security_solution/public/overview/components/user_overview/index.tsx

```diff
@@ -89,20 +88,9 @@ export const UserOverview = React.memo<UserSummaryProps>(
       [userName]
     );
 
-    const { from, to } = useGlobalTime();
-
-    const timerange = useMemo(
-      () => ({
-        from,
-        to,
-      }),
-      [from, to]
-    );
-
     const { data: userRisk, isAuthorized } = useRiskScore({
       filterQuery,
       skip: userName == null,
-      timerange,
```

Comment:
Why do we not pass a `timerange` to these hooks, anymore?

Reply:
Before the changes, we had a check that only used `timerange` if fetching the time series index (not the latest).

https://github.com/elastic/kibana/blob/dcc3f86c41bb5a22d0011112ae38aa5e4639dac4/x-pack/plugins/security_solution/public/explore/containers/risk_score/all/index.tsx#L171

Due to this check, the `timerange` provided inside the flyouts and details page was ignored. I deleted the check, so I needed to delete the properties otherwise, the flyouts and details page would also filter by time range.